omero/sysadmins/system-requirements.txt

```diff
@@ -62,38 +40,35 @@ With a large database and aggressive PostgreSQL caching your RAM usage
 could be larger. Still, even for a large deployment, it is not cost
 effective to use more than a few GBs of RAM for this purpose.
 :doc:`server-performance` provides information about fine-tuning the
-server processes' memory usage.
-
-Summary: Depending on hardware layout 16, 24 or 32GB of RAM would be
-ideal for your OMERO server. If you have a separate database server more
-than 16GB of RAM may not be of much benefit to you at all.
+server processes' memory usage. In summary, depending on hardware layout 16,
+24 or 32GB of RAM would be ideal for your OMERO server. If you have a separate
+database server more than 16GB of RAM may not be of much benefit to you at
+all.
 
 CPU is really not something that an OMERO system is almost ever limited
 by. However, when it is limited it is almost always limited by GHz and
 not by the CPU count. So you are not going to get a huge OMERO experience
-performance increase by, for example, throwing 24 cores at the problem.
-
-Summary: Depending on hardware layout 2 x 4, 2 x 6 system core count
-should be more than enough.
+performance increase by, for example, throwing 24 cores at the problem. In
+summary, depending on hardware layout 2 x 4, 2 x 6 system core count should be
+more than enough.
 
+:doc:`server-setup-examples` provides details on some production set-ups
+currently in use by OMERO admins, along with how many users and the amount of
+data they support, which you may find helpful.
 
 OMERO.insight and OMERO.importer
```

Comment:
JM says it is staying for 5.1 and no firm decision has been made about removing it for 5.2 yet either, so I'll leave it for now.